tfhe/src/core_crypto/fft_impl/common.rs

```diff
@@ -25,7 +25,7 @@ pub fn pbs_modulus_switch<Scalar: UnsignedTorus + CastInto<usize>>(
     // Start doing the right shift
     output >>= Scalar::BITS - poly_size.log2().0 - 2 + lut_count_log.0;
     // Do the rounding
-    output += output & Scalar::ONE;
+    output += Scalar::ONE;
```

Comment:
Is that strictly equivalent in that context ?

Reply:
it should be, I'll put a small thing together to explain why

Reply:
basically, output is always an odd number ?

Reply:
not really, I'll explain